src/backend/main/py.main/caliopen_main/message/parameters/draft.py

```diff
@@ -90,13 +86,19 @@ def _add_from_participant(self, user_id):
         user = User.get(user_id)
         if not hasattr(self, 'participants'):
             self.participants = []
+        else:
+            if len(self.participants) > 0:
+                for i, participant in enumerate(self.participants):
+                    if re.match("[Ff][Rr][Oo][Mm]", participant.type):
```

Comment:
a re.match("from", participant.type, re.I) should be more lisible (re.I for ignore case option)